phpunit.xml

```diff
@@ -21,8 +21,8 @@
         <env name="APP_ENV" value="testing"/>
         <env name="BCRYPT_ROUNDS" value="4"/>
         <env name="CACHE_DRIVER" value="array"/>
-        <!-- <env name="DB_CONNECTION" value="sqlite"/> -->
-        <!-- <env name="DB_DATABASE" value=":memory:"/> -->
+        <env name="DB_CONNECTION" value="sqlite"/> 
+        <env name="DB_DATABASE" value=":memory:"/>
```

Comment:
```suggestion
        <env name="DB_CONNECTION" value="testing_mysql"/> 
        <!-- <env name="DB_DATABASE" value=":memory:"/> -->
```

make separate the database for test, otherwise actual database value wipe out